Store/DataStoreModule.py

```diff
@@ -534,3 +579,32 @@ def getTabletypeData(self, tabletypes):
             for table in self.metaClasses[tabletype]:
                 retmap[table.__name__] = self.session.query(table).count()
         return retmap
+
+    # Populate methods in order to import CSV files
+
+    def populateReference(self, reference_data_folder=None):
+        """Import given CSV file to the given reference table"""
+        if reference_data_folder is None:
+            reference_data_folder = os.path.join("..", "default_data")
+
+        files = os.listdir(reference_data_folder)
+        reference_files = [file for file in files if file[:-4] in REFERENCE_TABLES]
+        for file in reference_files:
+            table_name = file[:-4]
+            possible_method = 'add' + table_name
```

Comment:
aah, now I understand why `addNationality` was renamed to `addNationalities`.

Hmm, that's a tricky one.  The method does only add a single new row, not multiple ones.

Aah,  got it. If you call the method(s) `addToNationalities`. By adding the `To` in the middle it still can infer that it's adding a single row.

Reply:
Add methods renamed.

https://github.com/debrief/python-import-library/pull/15/commits/29b87d2cc9d16d11caa51380761e8739f57550e1